x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -158,12 +161,13 @@ describe('AllCasesListGeneric', () => {
     useGetCasesMock.mockReturnValue(defaultGetCases);
     useGetTagsMock.mockReturnValue({ data: ['coke', 'pepsi'], isLoading: false });
     useGetCategoriesMock.mockReturnValue({ data: ['twix', 'snickers'], isLoading: false });
-    useGetCurrentUserProfileMock.mockReturnValue({ data: userProfiles[0], isLoading: false });
+    useGetCurrentUserProfileMock.mockReturnValue({ data: userProfiles[0].uid, isLoading: false });
```

Comment:
The `useGetCurrentUserProfile` returns the whole user profile and not only the `uid`. Why did this change fix the test?

Reply:
I was trying out stuff and forgot about this one, is not related and I will remove it, thanks for catching.